src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1829,7 +1830,7 @@ namespace winrt::TerminalApp::implementation
 
         try
         {
-            TerminalApp::TerminalSettings controlSettings;
+            TerminalSettings controlSettings;
```

Comment:
This constructs an empty `TerminalSettings`. Is that intended? Make sure it's intended.

Reply:
Looks like it isn't. I've changed it to `TerminalSettings controlSettings{nullptr}`.

This should've been happening before though so the change shouldn't do anything (other than avoiding the construction of an unused TerminalSettings). Right?

Reply:
Be very, very sure none of the control flows below this initialization relied on it not being null. I can imagine a world where the conditionals _fail_ but then we initialize a terminal with the "default settings".

Reply:
As long as it is provable that there is no codepath that tries to initialize with a null settings, I am okay accepting this.